src/MorphoAaveV3Adapter.ts

```diff
@@ -101,7 +99,7 @@ export class MorphoAaveV3Adapter extends MorphoAaveV3DataEmitter {
         shortDelay
       ),
       new StaticUserFetcher(
-        ADAPTER_MOCK.ethBalance,
+        ADAPTER_MOCK.userData,
```

Comment:
userData is now an object containing some data about the user, such as eth balance, stEth (non wrapped) information